etna/spec/controller_spec.rb

```diff
@@ -54,7 +54,7 @@ class Server < Etna::Server; end
       @app = setup_app(
         Arachne::Server,
         [ Etna::TestAuth ],
-        test: { log_file: @log_file }
+        test: { log_file: @log_file, hmac_keys: {etna: 'key'} }
```

Comment:
Because of the call to `super` in `Etna::TestAuth#valid?`, we now have to provide an `hmac_keys` config for tests so that `Etna::Auth#valid?` does not throw an exception.